Romo/lib/Drive/RomoNetworking/RMAddress.m

```diff
@@ -132,7 +132,7 @@ - (AddressInfo *)infoWithType:(SocketType)type
         hints.ai_flags = AI_PASSIVE;
         hints.ai_socktype = type;
         hints.ai_protocol = type == SOCK_STREAM ? IPPROTO_TCP : IPPROTO_UDP;
-        hints.ai_family   = PF_UNSPEC;
+        hints.ai_family   = AF_INET;
```

Comment:
Was this change needed?

Reply:
Without this, the address could get resolved to IPv6, and an error would occur from socket sendto function. By changing to AF_INET, we are forcing the address to use IPv4.

I guess before some iOS version, IPv4 was default so PF_UNSPEC was essentially same as AF_INET.